tasks/options/axe-webdriver.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+	firefox: {
```

Comment:
Adding chrome will crash travis ?. Maybe we could have two tests, one for travis and another with chrome to run locally.